### PR TITLE
Feature/cylinder texture

### PR DESCRIPTION
**Pull request by dabeshouse**

### CODE REVIEW

File: core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrDisc.java [hunk 1]
```diff
@@ -100,7 +138,7 @@ protected void pickGeometry(PickContext pc, boolean isSubElementRequired) {
       name = -1;
     }
     pc.gl.glPushName(name);
-    glDisc(pc);
+    glDisc(pc, false);
```
Comment (weijchen) [inline core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrDisc.java:141]: Is this supposed to be `glDisc(pc, true);` ?

File: core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrCylinder.java [hunk 2]
```diff
@@ -135,26 +186,31 @@ protected void pickGeometry(PickContext pc, boolean isSubElementRequired) {
       name = -1;
     }
     pc.gl.glPushName(name);
-    glCylinder(pc);
+    glCylinder(pc, false);
```
Comment (weijchen) [inline core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrCylinder.java:186]: also, the second argument should be true?

Comment (dabeshouse) [reply core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrCylinder.java:189]: This in `pickGeometry` that is used to determine interactions with the scene, so the full rendering is not required. In `renderGeometry` the isTextureEnabled does get checked and used (and is probably no longer needed above on line 125)

File: core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrDisc.java (hunk 1 above)
Comment (dabeshouse) [reply core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrDisc.java:141]: This in pickGeometry that is used to determine interactions with the scene, so the full rendering is not required. In renderGeometry the isTextureEnabled does get checked and used (and is probably no longer needed above on line 125)

File: core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrCylinder.java (hunk 2 above)
Comment (dabeshouse) [reply core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrCylinder.java:186]: Again, pickGeometry not renderGeometry, but this time I did remove the isTextureEnabled check in renderGeometry.

**Pull request merged**